Library/Homebrew/formula.rb

```diff
@@ -143,9 +148,10 @@ class Formula
   attr_accessor :build
 
   # @private
-  def initialize(name, path, spec)
+  def initialize(name, path, spec, install_name: name)
```

Comment:
Doesn't look like it's possible to use hash rockets to declare a keyword argument.